src/style-installer.js

```diff
@@ -110,12 +111,20 @@ export function insertStyleElement(doc, cssRoot, cssText, isRuntimeCss, ext) {
  *     be blocked on key services being loaded.
  */
 export function makeBodyVisible(doc, opt_waitForServices) {
+  const win = doc.defaultView;
+  const shouldMakeBodyBlock = isExperimentOn(win, 'make-body-block');
+  const bodyStyles = {
+    opacity: 1,
+    visibility: 'visible',
+    animation: 'none',
+  };
+
+  if (shouldMakeBodyBlock) {
+    bodyStyles['display'] = 'block';
```

Comment:
This is unfortunately too late for this. Let's do this in `viewport-impl.js` on top level somewhere.